omdrivers/lifecycle/iDRAC/iDRACJobs.py

```diff
@@ -147,6 +147,7 @@ def get_job_status(self, jobid):
         jobret = {"Status": TypeHelper.resolve(JobStatusEnum.InProgress)}
         if jobid.startswith('DCIM_OSD'):
             # Poll for OSD Concrete Job
+            time.sleep( 10 )

```

Comment:
```suggestion
```

you can delete this sleep time and add it in the calling section

Reply:
I have updated as you suggested.  Thanks @rajeevarakkal